### ISSUE TITLE
Export location is not accessible

### ISSUE THREAD

**Issue by oldman63** [Podcast-Republic/PodcastRepublic-iOS#31]
Successful export, but the save location is not accessible

file:///var/mobile/Containers/Data/Application/ only accessible hacked (Jailbreak) system

My suggestion: backup via the Files application

![IMG_0138-2](https://github.com/Podcast-Republic/PodcastRepublic-iOS/assets/3171626/619cafa7-659b-4f6a-b2fb-a38619822ac4)

**Comment by gemiren** [comment]
On iOS, this app can only export to folders of this app. You can open the "Files app > On my iPhone > Podcast Republic > Output" to access the exported data.

**Comment by oldman63** [comment]
thx.

This information should also be displayed in the application. It would be more useful than the message you see now.

**Comment by gemiren** [comment]
You are right. We don't need to display the full path in the iOS app. Will update that. Thanks!
